contracts/oraclize/MarketContractOraclize.sol

```diff
@@ -134,18 +111,20 @@ contract MarketContractOraclize is MarketContract, usingOraclize {
 
     /// @dev call to oraclize to set up our query and record its hash.
     function queryOracle() private {
-        if (oraclize_getPrice(ORACLE_DATA_SOURCE, QUERY_CALLBACK_GAS) > this.balance) {
-            OracleQueryFailed();
-            lastPriceQueryResult = "FAILED"; //TODO: failsafe
-        } else {
-            OracleQuerySuccess();
-            bytes32 queryId = oraclize_query(
-                ORACLE_QUERY_REPEAT,
-                ORACLE_DATA_SOURCE,
-                ORACLE_QUERY,
-                QUERY_CALLBACK_GAS
-            );
-            validScheduledQueryIDs[queryId] = true;
-        }
+        // Require that sufficient funds are available to pay for the query.
+        require(oraclize_getPrice(ORACLE_DATA_SOURCE, QUERY_CALLBACK_GAS) < this.balance);
+        // Require expiration time in the future.
+        require(EXPIRATION > now);
+        // Future timestamp must be within 60 days from now.
+        // https://docs.oraclize.it/#ethereum-quick-start-schedule-a-query-in-the-future
+        require(EXPIRATION - now <= 60 * 60 * 24 * 60);
```

Comment:
thoughts on using a constant variable for the 60 days value converted to seconds?

Reply:
Fine. I will create a variable for that.